dotcom-rendering/src/web/components/StickyVideo.importable.tsx

```diff
@@ -0,0 +1,91 @@
+import { css } from '@emotion/react';
+import { useEffect, useRef, useState } from 'react';
+import { getZIndex } from '../lib/getZIndex';
+import { useHasBeenSeen } from '../lib/useHasBeenSeen';
+
+const buttonStyles = css`
+	position: absolute;
+	left: 0;
+	top: 0;
+	${getZIndex('sticky-video-button')};
+`;
+
+const stickyStyles = (right: number) => css`
+	@keyframes fade-in-up {
+		from {
+			transform: translateY(100%);
+			opacity: 0;
+		}
+
+		to {
+			transform: translateY(0%);
+			opacity: 1;
+		}
+	}
+
+	position: fixed;
+	right: ${right}px;
```

Comment:
Just a thought, could you use flex here and just centre the contents? I did [something similar for the liveblog toast](https://github.com/guardian/dotcom-rendering/blob/19e6d6191bf26ea6703f04c74fa081bb579923e5/dotcom-rendering/src/web/components/Toast.tsx#L24) and that might be a lot simpler?

Reply:
I think the reason that approach might not work is that per the designs it needs to be aligned with the right margin of the central column, rather than centred. Unless you can think of a way of achieving that using `flex`?

The figma file is here: https://www.figma.com/file/K0kx2NX19inNJrUEcov8En/Pinned-video?node-id=271%3A2482

Reply:
Resolved using flexbox.